modules/utxo-lib/src/bitgo/dash/DashTransactionBuilder.ts

```diff
@@ -21,7 +21,7 @@ export class DashTransactionBuilder extends UtxoTransactionBuilder<DashTransacti
     this.tx.extraPayload = extraPayload;
   }
 
-  static fromTransaction(
+  static fromTransactionDash(
```

Comment:
Method removal. Please record as BREAKING CHANGE in commits.

Reply:
👍  refactoring